tools/generate_tests.py

```diff
@@ -65,3 +65,11 @@ def generate_tests(filename = 'testCases.json'):
 
             # Alignment check footer
             f.write(struct.pack('<4s', TEST_CASE_FOOTER))
+
+if __name__ == '__main__':
+    import argparse
```

Comment:
```suggestion
```

I'm not personally a fan of having import statements nested in between code. Can we move this to the top with `from argparse import ArgumentParser`?